book/website/communication/peer-review/peer-review-code.md

```diff
@@ -0,0 +1,91 @@
+(cm-pr-code)=
+
+# Guidance on Code Review
+
+Code review can help increase the accuracy of results, improve usability and maintainability of code, and is a great opportunity to learn. For a more detailed overview of motivations for Code Review, please see the {ref}`Code Reviewing Process <rr-reviewing-motivation>`.
+
+The most important questions during code review are: 
+- Are all the files available? If not, ask the editor to request the data/code from the authors.
+- Does it run? 
+- Is it easy to understand? 
+Or is it more complex than it should be?
+- If it is not just a script underlying a publication but part of software infrastructure: how easy is it to maintain? 
+
+[Code does not have to be perfect](https://google.github.io/eng-practices/review/reviewer/standard.html) - it has to work, be accompanied with sufficient documentation and be maintainable if this is needed.
+
+[Code review can take place asynchronous or synchronous](https://uwescience.github.io/neuroinformatics/2017/10/08/code-review.html). 
+Sometimes face-to-face communication (synchronous) is easier to address any questions that arise during the review {cite:ps}`Petre2014codereview`. 
```

Comment:
This is very true, but if I understand correctly this is about peer review. Do you intend here to contact the author for face-to-face reviewing?

Reply:
Good point.. this bit was more relevant when there was more into this chapter about software review for when it is a primary output or when you're collaborating in a lab - but that information is elsewhere. So I think i'll just delete that here to avoid confusion :)